autosubmitconfigparser/config/basicconfig.py

```diff
@@ -72,6 +71,24 @@ def props(self):
     DATABASE_BACKEND = "sqlite"
     DATABASE_CONN_URL = ""
 
+    @staticmethod
+    def EXPID_DIR(exp_id):
+        if not isinstance(exp_id, str) or len(exp_id) != 4:
+            raise TypeError("Experiment ID must be a string of 4 characters")
+        return os.path.join(BasicConfig.LOCAL_ROOT_DIR, exp_id)
+
+    @staticmethod
+    def EXPID_TMP_DIR(exp_id):
+        return os.path.join(BasicConfig.EXPID_DIR(exp_id), BasicConfig.LOCAL_TMP_DIR) 
+
+    @staticmethod
+    def EXPID_LOG_DIR(exp_id):
+        return os.path.join(BasicConfig.EXPID_TMP_DIR(exp_id), f'LOG_{exp_id}')
+
+    @staticmethod
+    def EXPID_ASLOG_DIR(exp_id):
```

Comment:
For functions it's normally lower case, kebab style in Python, whereas variables with capital letters are normally constants.

Reply:
Sure! I thought it would be good to match the attribute's style (`BasicConfig.LOCAL_ROOT_DIR`), but I'll chage it + add tests